packages/sdk/src/lib/Signer.ts

```diff
@@ -43,6 +53,7 @@ export const sign: (
 
 const throwIfNotConnected = () => {
   if (!isConnected()) {
+    alert('NOT CONNECTED');
```

Comment:
@AbnerZheng Is there a better way of displaying such messages?

Reply:
I think this may have been used for debugging rather than for users - I'll remove it now

Reply:
It would throw an error, and end-users like Clarity show the error in the front end.